cookbooks/cdo-varnish/README.md

```diff
@@ -1,68 +1,50 @@
-cdo-varnish Cookbook
-====================
-TODO: Enter the cookbook description here.
-
-e.g.
-This cookbook makes your favorite breakfast sandwich.
-
-Requirements
-------------
-TODO: List your cookbook requirements. Be sure to include any requirements this cookbook has on platforms, libraries, other cookbooks, packages, operating systems, etc.
-
-e.g.
-#### packages
-- `toaster` - cdo-varnish needs toaster to brown your bagel.
-
-Attributes
-----------
-TODO: List your cookbook attributes here.
-
-e.g.
-#### cdo-varnish::default
-<table>
-  <tr>
-    <th>Key</th>
-    <th>Type</th>
-    <th>Description</th>
-    <th>Default</th>
-  </tr>
-  <tr>
-    <td><tt>['cdo-varnish']['bacon']</tt></td>
-    <td>Boolean</td>
-    <td>whether to include bacon</td>
-    <td><tt>true</tt></td>
-  </tr>
-</table>
-
-Usage
------
-#### cdo-varnish::default
-TODO: Write usage instructions for each cookbook.
-
-e.g.
-Just include `cdo-varnish` in your node's `run_list`:
-
-```json
-{
-  "name":"my_node",
-  "run_list": [
-    "recipe[cdo-varnish]"
-  ]
-}
-```
-
-Contributing
-------------
-TODO: (optional) If this is a public cookbook, detail the process for contributing. If this is a private cookbook, remove this section.
-
-e.g.
-1. Fork the repository on Github
-2. Create a named feature branch (like `add_component_x`)
-3. Write your change
-4. Write tests for your change (if applicable)
-5. Run the tests, ensuring they all pass
-6. Submit a Pull Request using Github
-
-License and Authors
--------------------
-Authors: TODO: List authors
+# cdo-varnish Cookbook
+Installs and configures Varnish HTTP cache.
+
+## Requirements
+Ubuntu 14.04
+
+#### apt packages installed (from PPA)
+- `varnish`
+- `libvmod-cookie`
+- `libvmod-header`
+
+## `HttpCache#config`
+
+Provides application-specific cache configuration used by Varnish and other
+HTTP cache layers.
+`pegasus` and `dashboard` keys each return a Hash in the following format:
+- `behaviors`: Array of behaviors, evaluated in top-to-bottom order:
+  - `path`: [Path pattern](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-values-specify.html#DownloadDistValuesPathPattern)
+    to match this behavior against.
+    `*`-wildcards are allowed, either extension-wildcard `*.jpg` or
```

Comment:
I think the link to the Amazon documentation is potentially misleading.  It includes example paths which I believe aren't supported here (e.g. /images/*.jpg and /a??.jpg)